create/how_to/updating.md

```diff
@@ -233,15 +242,16 @@ Once the response to the previous command looks like this (`"status": "done"`),
 }
 ```
 
-### Step 3: Import the dump
+### Step 3: Delete the database folder
+
+To delete the old MeiliSearch version, you need to delete data.ms. data.ms should be at the root of the MeiliSearch binary, unless you chose [another location](https://docs.meilisearch.com/reference/features/configuration.html#database-path).
```

Comment:
Put data.ms between backticks so it doesn't create a link

Reply:
@CaroFG 
data.ms is used without `` throughout the documentation. Should I update that everywhere?

Reply:
I think it should because the link redirects to a website that has nothing to do with us, but this can be done in another PR. You can create an issue so you don't forget